cli/beamable.common/Runtime/Modules/SharedContent/SimGameType.cs

```diff
@@ -140,6 +153,14 @@ public void OnAfterDeserialize()
 				};
 			}
 
+			if (federatedGameServerNamespace.HasValue && !newFederatedGameServerNamespace.HasValue && isFirstTime)
+			{
+				isFirstTime = false;
+				newFederatedGameServerNamespace.HasValue = true;
+				var nameSpace = new Namespace {Name = federatedGameServerNamespace};
+				newFederatedGameServerNamespace.Value = nameSpace;

```

Comment:
You would need to check but if I remember correctly there are implicit operator methods for that so you should be able to just do 
```suggestion
				newFederatedGameServerNamespace = new Namespace {Name = federatedGameServerNamespace};
```